scripts/setup.sh

```diff
@@ -186,6 +186,9 @@ if [ ! -d "$NODE_PATH" ]; then
   chmod -R 744 $NODE_MODULE_DIR
 fi
 
+# Change execute status of browsermob-proxy
+chmod +x $TOOLS_DIR/browsermob-proxy-2.1.1/bin/browsermob-proxy
```

Comment:
I'm really not keen on this. Can we find a workaround? Setting the executable bit of a file within ``setup.sh`` is sort of screaming to have a security incident with one of our developers, which we certainly don't want to happen. There might be a workaround where we only set the bit when running the performance tests and make sure to unset it. If we can avoid needing to do this altogether, it would definitely be preferred.